src/com/ean/clarity/Timesheet.java

```diff
@@ -0,0 +1,62 @@
+package com.ean.clarity;
+
+import java.util.Random;
+
+/**
+ * Represents a timesheet with a single task because why would you ever need to have multiple tasks?
+ */
+public class Timesheet {
+
+    private static final int VALID_HOURS = 8;
+
+    private int mondayHours;
+    private int tuesdayHours;
+    private int wednesdayHours;
+    private int thursdayHours;
+    private int fridayHours;
+
+    /**
+     * "Submits" the timesheet.
+     */
+    public void submit() {
+        int totallyArbitraryRandomNumber1 = new Random().nextInt(5);
+        int totallyArbitraryRandomNumber2 = new Random().nextInt(5);
+        if (totallyArbitraryRandomNumber1 != totallyArbitraryRandomNumber2) {
+            throw new UnsupportedOperationException();
+        }
+        // TODO: Pretend to submit the timesheet
```

Comment:
You know there is an actual API for clarity, we just don't have access to use it....maybe we should look into that?